be/src/common/config.h

```diff
@@ -265,6 +265,27 @@ namespace config {
     CONF_mInt64(base_compaction_interval_seconds_since_last_operation, "86400");
     CONF_mInt32(base_compaction_write_mbytes_per_sec, "5");
 
+    // config the cumulative compaction policy
+    // Valid configs: ordinary, universal
+    // ordinary policy, the original version of cumulative compaction, cumulative version compaction once.
+    // universal policy, a optimization version of cumulative compaction, targeting the use cases requiring 
+    // lower write amplification, trading off read amplification and space amplification.
+    CONF_String(cumulative_compaction_policy, "original");
+
+    // In universal policy, output rowset of cumulative compaction total disk size exceed this config size, 
+    // this rowset will be given to base compaction, unit is m byte.
+    CONF_mInt64(cumulative_compaction_universal_promotion_size_mbytes, "1024");
```

Comment:
These compaction config is not mutable at runtime because you init the compaction policy only once when initializing the tablet. So modify these configs makes no effect at runtime.
But I think we should find a way to make it mutable at runtime.